src/error/metadata/mod.rs

```diff
@@ -71,6 +71,7 @@ impl From<&mut anyhow::Error> for Metadata {
                 }
                 RoverClientError::InvalidKey => (Some(Suggestion::CheckKey), None),
                 RoverClientError::MalformedKey => (Some(Suggestion::ProperKey), None),
+                RoverClientError::UnparseableReleaseVersion => (None, None),
```

Comment:
seems like this should maybe be one of the "file an issue" suggestions, what do you think?